### ISSUE TITLE
Please document currently unsupported features

### ISSUE THREAD

**Issue by gordonfreeman01** [nba-emu/NanoBoyAdvance#304]
Love the emulator but it is currently unclear from the documentation which GBA/cartridge features are unsupported for 100% compatibility (e.g. tilt sensors in some games). Could those be specified in the documentation please?

**Comment by Felipefpl** [comment]
Indeed, it would be nice to know what is still unemulated so far. :+1:
